### PR TITLE
removal round 2

### PR DESCRIPTION
**Pull request by YueErro**

### CODE REVIEW

File: mara_description/scripts/mara_state_publisher.py [hunk 1]
```diff
@@ -1,209 +0,0 @@
-#!/usr/bin/python3
```
[the remark this answers is not in the document]
Comment (YueErro) [reply mara_description/scripts/mara_state_publisher.py]: Are you sure? 
I have tested and i can visualize the robot model in rviz and also the /joint_states topic, i can do echo of it and i see the values changing.

[the remark this answers is not in the document]
Comment (YueErro) [reply mara_description/scripts/mara_state_publisher.py]: Using this instead:
https://github.com/AcutronicRobotics/MARA/blob/master/mara_gazebo/launch/mara.launch.py#L49

**Comment by nzlz** [comment]
I think ive seen that demo file referenced somewhere in the docs, but i agree we should remove that. Ill update the docs if i find it.

Regarding the mara state publisher, isnt rviz fed using the robot state publisher and not this script? Ill take a deeper look tomorrow if this PR is not solved by then.

Other removals / mods look fine.

File: mara_description/scripts/mara_state_publisher.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (rkojcev) [reply mara_description/scripts/mara_state_publisher.py]: Can we try on a clean machine (docker) if this file is needed any longer? @alesolano ?

**Comment by rkojcev** [comment]
What is the status of this one @ahcorde? can we accept this?

File: mara_description/scripts/mara_state_publisher.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (alesolano) [reply mara_description/scripts/mara_state_publisher.py]: It works for me

**Comment by LanderU** [comment]
Can be merged?

**Comment by YueErro** [comment]
I would say yes.

**Pull request merged**